pylint/checkers/refactoring/refactoring_checker.py

```diff
@@ -464,9 +464,9 @@ class RefactoringChecker(checkers.BaseTokenChecker):
             "The literal is faster as it avoids an additional function call.",
         ),
         "R1735": (
-            "Consider using {} instead of dict()",
+            "Consider using a dict literal '%s' instead.",
```

Comment:
```suggestion
            "Consider using '%s' instead.",
```
Wondering if this would be better or not (I"m aware I suggested the initial message 😄).

Reply:
I'm always a fan of short and concise :+1: :slightly_smiling_face: